ogre2/src/Ogre2DynamicRenderable.cc

```diff
@@ -131,8 +137,8 @@ void Ogre2DynamicRenderable::DestroyBuffer()
   if (this->dataPtr->vbuffer)
     delete [] this->dataPtr->vbuffer;
 
-  Ogre::Root *root = Ogre2RenderEngine::Instance()->OgreRoot();
-  Ogre::RenderSystem *renderSystem = root->getRenderSystem();
+  Ogre::RenderSystem *renderSystem =
+      this->dataPtr->sceneManager->getDestinationRenderSystem();
```

Comment:
This change seems unrelated to reducing the headers, is it needed as part of this PR?

Reply:
Actually yes. Including OgreRoot.h is not necessary because RenderSystem can be obtained from another pointer

Reply:
Actually yes. Including OgreRoot.h is not necessary because RenderSystem can be obtained from another pointer